src/System.Security.Principal.Windows/tests/WindowsPrincipalTests.cs

```diff
@@ -33,4 +35,33 @@ public static void WindowsPrincipalIsInRoleNeg()
             Assert.Equal(win32Exception.Message, e.Message);
         }
     }
+
+    [Fact]
+    public static void CheckDeviceClaims()
+    {
+        using (WindowsIdentity id = WindowsIdentity.GetCurrent())
```

Comment:
GetCurrent can throw a SecurityException - does this test need to be hardened for running as non-admin

Reply:
Works fine as a non-admin.  The SidType tests have a precondition that already calls `WindowsIdentity.GetCurrent()`, so if there are situations where it fails we haven't had anyone encounter them and report it.  (Maybe there's an extraordinary set of restrictions for a sandboxed process where it can't know its ProcessHandle or ProcessToken, but I don't, personally, know of them)